src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,17 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery firstBall = new Lottery();
+        Lottery secondBall = new Lottery();
+        Lottery thirdBall = new Lottery();
+
+        Lottery[] lotteries = new Lottery[3];
+        lotteries[0] = firstBall;
+        lotteries[1] = secondBall;
+        lotteries[2] = thirdBall;
```

Comment:
Why do we need array of lotteries? We could have only one lottery object and call `getRandomBall()` on it for 3 times in a loop `for (int i = 0; i < 3; i++)`